lib/dry/web/roda/templates/container.rb.tt

```diff
@@ -1,5 +1,7 @@
 require "dry/web/container"
 require "dry/system/components"
+require 'dry-monitor'
+Dry::Monitor.load_extensions(:rack)
```

Comment:
a workaround..

Reply:
Yeah, but for me logically should be where we have a rack app so system/web ? Not sure it's possible to put there? Or maybe somehow on boot/monit ?

Reply:
@gotar it seems dry-rb/dry-web#46 exists for this.. I'm not sure where else we'd put it apart from in this container file.. everywhere else I try fails.

Reply:
Yep, I think the right approach here is to get https://github.com/dry-rb/dry-web/pull/46 finished and merged. So perhaps we can take this out now?

Reply:
I'll take this out once dry-rb/dry-web#46 is fixed because this makes my app work as I'm using this branch for it, to use the generators, so I'd like them to work :wink: